README.md

```diff
@@ -42,19 +42,19 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
 
 ## Install
 
-We suggest a couple of alternative ways of installation.
+We provide a couple of alternative ways of installation.
 
 **Instructions for installing bash, zsh, and fish completion scripts are provided in [completion/README.md](completion/README.md).**
 
 git-machete requires Python >= 3.6. Python 2.x is no longer supported.
 
-### Using Homebrew (macOS)
+### Using Homebrew (macOS & most Linux distributions)
 
 ```shell script
 brew install git-machete
 ```
 
-### Using Snappy (most Linux distributions)
```

Comment:
I've never seen this name being actually used for snap package manager... in fact this might've been a mistake to begin with. The most prominent thing called Snappy seems to be https://github.com/google/snappy